WordPressKit/Sources/WordPressKit/Models/Stats/Time Interval/StatsFileDownloadsTimeIntervalData.swift

```diff
@@ -22,11 +22,17 @@ public struct StatsFileDownloadsTimeIntervalData {
 public struct StatsFileDownload {
     public let file: String
     public let downloadCount: Int
+    public let relativeURL: String
```

Comment:
(nit) `relativeURL` is unused, so I'd suggest not adding an expectation that it's going to be there.